pubsub/topics/topics_test.go

```diff
@@ -151,11 +161,12 @@ func TestIAM(t *testing.T) {
 }
 
 func TestDelete(t *testing.T) {
-	c := setup(t)
-	if err := delete(c, topicID); err != nil {
+	tc := testutil.SystemTest(t)
+	setup(t)
+	if err := delete(tc.ProjectID, topicID); err != nil {
```

Comment:
Create a topic if it doesn't already exist before calling `delete`? Same as for subscriptions.